copter/source/docs/tuning-process-instructions.rst

```diff
@@ -117,27 +244,28 @@ Most pilots will look to move to Autotune as quickly as possible once their airc
 3. Increase inputs gradually to full stick deflection.
 4. Go to full stick deflection and letting the sticks spring back to centre.
 
-If the aircraft begins to overshoot significantly or oscillate, after the stick input halt the tests before the situation begins to endanger the aircraft. The aircraft may require manual tuning before autotune can be run.
+If the aircraft begins to overshoot significantly or oscillate after the stick input, halt the tests before the situation begins to endanger the aircraft. The aircraft may require manual tuning (:ref:`see next section <Manual tuning of Roll and Pitch>`) before autotune can be run.
 
-To test the stabilization loops independent of the input shaping, set the parameter: ATC_RATE_FF_ENAB to 0.
+To test the stabilization loops independent of the input shaping, set the parameter: :ref:`ATC_RATE_FF_ENAB <ATC_RATE_FF_ENAB>` to 0.
 
 1. Take off in Alt Hold or Stabilize
 2. Hold a roll or pitch input
 3. Release the stick and observe the overshoot as the aircraft levels itself.
 4. Gradually increase the stick deflection to 100%
 
-Halt the tests if the aircraft overshoots level significantly or if the aircraft oscillates, the aircraft may require manual tuning before autotune can be run.
+Halt the tests if the aircraft overshoots level significantly or if the aircraft oscillates, the aircraft may require manual tuning (:ref:`see next section <Manual tuning of Roll and Pitch>`) before autotune can be run.
 
-Set ATC_RATE_FF_ENAB to 1 after the tests are complete.
+Set :ref:`ATC_RATE_FF_ENAB <ATC_RATE_FF_ENAB>` to 1 after the tests are complete.
 
 Manual tuning of Roll and Pitch
 -------------------------------
 
 Manual tuning may be required to provide a stable tune before Autotune is run, or if Autotune does not produce an acceptable tune. The process below can be done on roll and pitch at the same time for a quick manual tune provided the aircraft is symmetrical. If the aircraft is not symmetrical then the process should be repeated for both roll and pitch individually.
 
-The pilot should be especially careful to ensure that ATC_THR_MIX_MAN and MOT_THST_HOVER are set correctly before manual tuning is started.
+The pilot should be especially careful to ensure that :ref:`ATC_THR_MIX_MAN <ATC_THR_MIX_MAN>` and :ref:`MOT_THST_HOVER <MOT_THST_HOVER>` are set correctly before manual tuning is started.
 
 When oscillations start do not make large or sudden stick inputs. Reduce the throttle smoothly to land the aircraft while using very slow and small roll and pitch inputs to control the aircraft position.
+# todo : in flight ? in hover state ?
```

Comment:
This is true for any state so I don't think we need to elaborate.

Reply:
So we set the parameter when land , right ?

Reply:
Again, there are a number of ways to do this. I am happy to leave it up to the pilot.